pkg/liqo-controller-manager/storageprovisioner/remoteprovisioner.go

```diff
@@ -97,12 +97,19 @@ func ProvisionRemotePVC(ctx context.Context,
 }
 
 // remotePersistentVolumeClaim forges the apply patch for the reflected PersistentVolumeClaim, given the local one.
-func remotePersistentVolumeClaim(virtualPvc *corev1.PersistentVolumeClaim,
+func remotePersistentVolumeClaim(virtualPvc *corev1.PersistentVolumeClaim, remotePvc *corev1.PersistentVolumeClaim,
 	storageClass, namespace string) *v1apply.PersistentVolumeClaimApplyConfiguration {
-	return v1apply.PersistentVolumeClaim(virtualPvc.Name, namespace).
+	applyConfigurations := v1apply.PersistentVolumeClaim(virtualPvc.Name, namespace).
 		WithLabels(virtualPvc.GetLabels()).
 		WithLabels(forge.ReflectionLabels()).
+		WithAnnotations(filterAnnotations(virtualPvc.GetAnnotations())).
 		WithSpec(remotePersistentVolumeClaimSpec(virtualPvc, storageClass))
+	if remotePvc != nil {
+		return applyConfigurations.
+			WithAnnotations(remotePvc.GetAnnotations()).
+			WithLabels(remotePvc.GetLabels())
+	}
```

Comment:
It is not required to explicitly add the remote labels/annotations, since only managed fields are modified when using server side apply.